lib/src/scss-shared/_nx-focus-border-helpers.scss

```diff
@@ -6,12 +6,12 @@
  */
 
 // For dynamically calculating focus border styles based on height and width
-// Border size is 2px border and 4px here represents total space in each axis
+// Border size is 2px border + 2px gap = 2 * 4px here to represents total space in each axis
```

Comment:
This seems like it would affect light mode too wouldn't it?

Reply:
> This seems like it would affect light mode too wouldn't it?

I think in light mode it is also 2px for focus borders.

Reply:
> This seems like it would affect light mode too wouldn't it?

Oh yeah, this was also one of the reasons why I was trying out box-shadow, it simplified the left, top calculation on the ::after

Reply:
> I think in light mode it is also 2px for focus borders.

Yes but I was trying to understand how this change doesn't result in a difference to light mode.  I now see that this is essentially cancelled out by the `box-sizing: border-box` that's applied to the `::after` in this PR.  I suspect you could get rid of both that change and this change together with no visual difference.

Reply:
> Oh yeah, this was also one of the reasons why I was trying out box-shadow, it simplified the left, top calculation on the ::after

OK that's a decent reason.  Please document that in a code comment near the point where the box-shadow stuff is implemented.

Reply:
> > Oh yeah, this was also one of the reasons why I was trying out box-shadow, it simplified the left, top calculation on the ::after
> 
> OK that's a decent reason. Please document that in a code comment near the point where the box-shadow stuff is implemented.

Added Comment here: 64a881d4a98b35a73759f8e0e3ebb680923d9562